프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js

```diff
@@ -0,0 +1,6 @@
+function solution(price, money, count) {
+    const amountPerUse = Array(count).fill().map((_, idx) => (idx + 1) * price);
+    const amountPrice = amountPerUse.reduce((acc, cur) => acc + cur, 0);
+    const shortFall = money - amountPrice;
+    return shortFall > 0 ? 0 : Math.abs(shortFall);
```

Comment:
```suggestion
    const shortfall = money - totalCost;
    return shortfall > 0 ? 0 : Math.abs(shortfall);
```
위에을 토탈로 계산한다면 밑에를 요로케 바꿔주면 더 간단한 풀이가 되지 않을까요 ~?